index.html

```diff
@@ -29,22 +30,205 @@ <h1>
+            <li><a href="#">CSS</a></li>
+            <li><a href="#">JavaScript</a></li>
+            <li><a href="#">HTML</a></li>
+        </ul>
+        <div class="button"><a href="#">See project</a></div>
+      </div>
+      </div>
+<!--3-->
+      <div class="item">
+        <div class="border">
+          <div class="border-image">
+          <img src="images/grid.svg" alt="">
+        </div>
+        </div>
+        <div class="content">
+          <div class="title"><p>
+            Multi-Post Stories <br>Gain+Glory
+            </p></div> 
+        <ul class="links">
+          <li><a href="#">Ruby on Rails</a></li>
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">JavaScript</a></li>
+          <li><a href="#">HTML</a></li>
+      </ul>   
+      <div class="button"><a href="#">See project</a></div>
+    </div>
+
+<!--4-->
+    <div class="item">
+      <div class="border">
+        <div class="border-image">
+        <img src="images/grid.svg" alt="">
+      </div>
+      </div>
+      <div class="content">
+        <div class="title"><p>
+          Multi-Post Stories <br>Gain+Glory
+          </p></div>
+      <ul class="links">
+        <li><a href="#">Ruby on Rails</a></li>
+        <li><a href="#">CSS</a></li>
+        <li><a href="#">JavaScript</a></li>
+        <li><a href="#">HTML</a></li>
+    </ul>
+    <div class="button"><a href="#">See project</a></div>
+  </div>
+
+<!--5-->
+  <div class="item">
+    <div class="border">
+      <div class="border-image">
+      <img src="images/grid.svg" alt="">
+    </div>
+    </div>
+    <div class="content">
+      <div class="title"><p>
+        Multi-Post Stories <br>Gain+Glory
+        </p></div>
+    <ul class="links">
+      <li><a href="#">Ruby on Rails</a></li>
+      <li><a href="#">CSS</a></li>
+      <li><a href="#">JavaScript</a></li>
+      <li><a href="#">HTML</a></li>
+  </ul>
+  <div class="button"><a href="#">See project</a></div>
+</div>
+
+<!--6-->
+<div class="item">
+  <div class="border">
+    <div class="border-image">
+    <img src="images/grid.svg" alt="">
+  </div>
+  </div>
+  <div class="content">
+    <div class="title"><p>
+      Multi-Post Stories <br>Gain+Glory
+      </p></div>
+  <ul class="links">
+    <li><a href="#">Ruby on Rails</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">JavaScript</a></li>
+    <li><a href="#">HTML</a></li>
+</ul>
+<div class="button"><a href="#">See project</a></div>
+</div>
+</div>
+</section>
+
+<!--About-->
+<section class="about">
+  <div class="about-me-text">
+      <h2 class="about-text">About me</h2>
+      <div class="about-heading">I can help you build a product , <br>feature or website 
+        Look through <br>some of my work and
+         experience! If <br>you like what
+          you see and have a <br>project you
+           need coded,<br> don’t hesitate to contact me.</div>
+      <div class="resume_button"><a href="">Get my resume</a></div>
```

Comment:
- [ ] Great job using the correct tag for the `Get my resume` button but it seems you didn't implement the different interactive state of the button, ill suggest you do this as its part of the requirement for this project: 🛩️ 💯 
> Implement the button interactions (enable, hover, etc.).